### PR TITLE
Add holding content for bursaries and scholarships

### PR DESCRIPTION
**Pull request by emmaachesongray**
### Trello card
https://trello.com/c/K2dna8p2

### CODE REVIEW

File: app/views/content/life-as-a-teacher/explore-subjects/biology/_article.html.erb [hunk 1]
```diff
@@ -96,14 +96,15 @@
 
   <section class="col col-space-m col-720">
     <h3>Fund your teacher training</h3>
+    <%= render Content::InsetTextComponent.new(text: "The figures on this page apply if you're starting your training $bursaries_fundingdates$. Financial support for courses starting the following year will be announced in autumn 2024.") %>
     <p><a href="/funding-and-support/scholarships-and-bursaries">Tax-free bursaries of <%= v :bursaries_postgraduate_biology %> are available for eligible trainee biology teachers</p>
```
Comment (gemmadallmandfe) [inline app/views/content/life-as-a-teacher/explore-subjects/biology/_article.html.erb:100]: Nothing to do with holding message but there is no closing `</a>` and it is missing a full-stop outside the link text - this needs updating on more than one subject page so suggest we do this retrospectively as we do other subject page changes

cc @Sarah-DfE to add to the subject page ticket!

**Comment by gemmadallmandfe** [comment]
@emmaachesongray Need to add similar content to the ETP page `/life-as-a-teacher/explore-subjects/engineers-teach-physics`

File: app/views/content/funding-and-support/scholarships-and-bursaries/_funding-widget.html.erb [hunk 2]
```diff
@@ -21,7 +21,9 @@
     link_title: "Learn more about fees and financial support for non-UK trainee teachers.",
     link_url: "/non-uk-teachers/fees-and-funding-for-non-uk-trainees",
   ) %>
-    <p>The figures on this page apply if you're starting your training <%= value :bursaries_fundingdates %>.</p>
+  </section>
+  <section>
+    <%= render Content::InsetTextComponent.new(text: "The figures on this page apply if you're starting your training $bursaries_fundingdates$. Financial support for courses starting the following year will be announced in autumn 2024.") %>
```
Comment (gemmadallmandfe) [inline app/views/content/funding-and-support/scholarships-and-bursaries/_funding-widget.html.erb:26]: @emmaachesongray I wonder if we should put this inset component at the top of the page in this interim period - pretty sure that is what we did last year

and then when the new bursaries are out, we can go back to having the date reference here

**Comment by emmaachesongray** [comment]
@gemmadallmandfe all updated - please can you review again?

**Pull request merged**